packages/mdc-tab-scroller/rtl-scroller.js

```diff
@@ -33,8 +33,8 @@ import {MDCTabScrollerAdapter, MDCTabScrollerAnimation} from './adapter';
 class MDCTabScrollerRTL {
   /** @param {!MDCTabScrollerAdapter} adapter */
   constructor(adapter) {
-    /** @private */
-    this.adapter_ = adapter;
+    /** @package {!MDCTabScrollerAdapter} */
```

Comment:
I forget, does closure not have `@protected`? (We're changing this to `protected readonly` on the TypeScript branch.)

Reply:
`@protected` sounds relevant here.